src/components/structures/ScrollPanel.js

```diff
@@ -148,6 +148,7 @@ module.exports = React.createClass({
             onFillRequest: function(backwards) { return Promise.resolve(false); },
             onUnfillRequest: function(backwards, scrollToken) {},
             onScroll: function() {},
+            onResize: function() {},
```

Comment:
Is this still useful to add? Seems like it might be

Reply:
Oh. I had left it in unintentionally. I think it is useful; things might not need to do anything if the SP changes size.